### DISCUSSION TITLE
lib32-vulkan-driver

### DISCUSSION THREAD

**Discussion by dieBakterie** [prasanthrangan/hyprdots#1181] [Q&A] [answered]
Should i select the `lib32-nvidia-utils` one because i've an Nvidia gpu?!

**Comment by kRHYME7** [comment, marked as answer]
lib32-nvidia-utils is needed when you have a 64-bit Linux system and need to run 32-bit applications that require GPU acceleration

**Comment by dieBakterie** [comment]
Ok thanks for the fast help and answer
